test/SourceKit/CursorInfo/cursor_groups.swift

```diff
@@ -0,0 +1,85 @@
+// RUN: %empty-directory(%t)
+// RUN: mkdir -p %t/mods
+// RUN: split-file --leading-lines %s %t
```

Comment:
(Not related to this PR)
Oh, I didn't know `split-file` utility, it's new from `llvm-project`. We have `utils/split_file.py` (`RUN: %{python} %utils/split_file.py`). I think we should use `split-file` consistently :)

Reply:
Yeah, it's on my todo list to change everything over to use `split-file`. It's better in a couple ways, ie. automatically creates any directories and allows leading-lines. Hopefully I'll get to it at some point 😅